loader/loader.go

```diff
@@ -89,7 +89,7 @@ var versionWarning []string
 
 func (o *Options) warnObsoleteVersion(file string) {
 	if !slices.Contains(versionWarning, file) {
-		logrus.Warning(fmt.Sprintf("%s: `version` is obsolete", file))
+		logrus.Warning(fmt.Sprintf("%s: the attribute `version` is deprecated,it will be ignored, please remove it to avoid potential confusion", file))
```

Comment:
version is not deprecated. If we want to, need first to update https://github.com/compose-spec/compose-spec/blob/master/04-version-and-name.md.

Reply:
thanks, updated